index.html

```diff
@@ -0,0 +1,108 @@
+<!doctype html>
+<html>
+
+<head>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <link href="https://fonts.cdnfonts.com/css/new-york-extra-large" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-mono" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <meta charset="UTF-8">
+    <title></title>
+</head>
+
+<body class="body">
+    <div >
+        <div class="top">
+            <h1 class="ROSE"><img width="235px" height="30px" src="rose.png" alt=""></h1>
+            <div class="wholeBurger">
```

Comment:
classnames should be in kebab-case
```suggestion
            <div class="whole-burger">
```
I think I explained this on the call. why wasn't it done?